packages/notifications/src/Livewire/DatabaseNotifications.php

```diff
@@ -28,6 +29,8 @@ class DatabaseNotifications extends Component
 
     public static ?string $authGuard = null;
 
+    public static ?Action $markAllNotificationsAsReadAction = null;
```

Comment:
Instead of storing the action, please add `HasActions` and `InteractsWithActions` to this component, and then rename `getMarkAllNotificationsAsReadAction` to `markAllNotificationsAsReadAction`. Then you can use `$this->markAllNotificationsAsReadAction` to render

Reply:
YOu won't need this line anymore